### DISCUSSION TITLE
When specifying 'or' as the query parameter of a get request the sql shows an 'and' clause, why is this?

### DISCUSSION THREAD

**Discussion by atreeon** [supabase/supabase#8773] [Questions] [answered]
I'm unsure if this is a problem with PostgREST or Supabase (I understood that Supabase just uses PostgREST, I posted a link there but unsure where the problem lies.

https://github.com/PostgREST/postgrest/issues/2462

My get request is this
`employees?employee_id=gt.4&or=(employee_id.eq.6)`

but the sql (using supabase logging) is the following
`SELECT "public"."employees".*FROM "public"."employees" WHERE ( "public"."employees"."employee_id" = $1) AND "public"."employees"."employee_id" > $2`

I would have expected the AND to be an OR.

This of course produces an unexpected result. Am I misunderstanding how postgrest works?

(full sql is the following)
`WITH pgrst_source AS ( SELECT "public"."employees".*FROM "public"."employees" WHERE ( "public"."employees"."employee_id" = $1) AND "public"."employees"."employee_id" > $2 LIMIT $3 OFFSET $4 ) SELECT null::bigint AS total_result_set, pg_catalog.count(_postgrest_t) AS page_total, array[]::text[] AS header, coalesce(json_agg(_postgrest_t), '[]')::character varying AS body, nullif(current_setting('response.headers', true), '') AS response_headers, nullif(current_setting('response.status', true), '') AS response_status FROM ( SELECT * FROM pgrst_source ) _postgrest_t`

**Comment by GaryAustin1** [comment, marked as answer]
The .or() filter is only going to "or" two or more filter statements within the () separated by a comma.
Looks like your question is answered in a discussion link to your other issue...
https://github.com/PostgREST/postgrest/discussions/2463

**Comment by atreeon** [comment]
I got it!
